theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/AggregatedKeyframeTrack/AggregateKeyframeEditor/AggregateKeyframeVisualDot.tsx

```diff
@@ -5,7 +5,7 @@ import {absoluteDims} from '@theatre/studio/utils/absoluteDims'
 import {DopeSnapHitZoneUI} from '@theatre/studio/panels/SequenceEditorPanel/RightOverlay/DopeSnapHitZoneUI'
 
 const DOT_SIZE_PX = 16
-const DOT_HOVER_SIZE_PX = DOT_SIZE_PX + 5
+const DOT_HOVER_SIZE_PX = DOT_SIZE_PX + 2
```

Comment:
Changing the hover size was requested and approved by Mariusz. ([link to Slack message](https://theatrejs.slack.com/archives/C031ZL87KB4/p1654692490031659?thread_ts=1654687856.597119&cid=C031ZL87KB4))